File: src/assets/usage-examples-code/color/color_purple/color_purple-1-example.cpp
#include "splashkit.h"

int main()
{
    // Define the color Purple
    color my_color = COLOR_PURPLE;

    // Open a window
    open_window("Purple Example", 400, 400);

    // Draw a filled circle on the screen using the color from above
    clear_screen();
    fill_circle(my_color, 200, 200, 100);
    refresh_screen();

    // Keep the window open for 5 seconds
    delay(5000);

    // Close the window
    close_window("Purple Example");

    return 0;
}

File: src/assets/usage-examples-code/color/color_purple/color_purple-1-example.cs
using SplashKitSDK;

// Define the color Purple
Color myColor = SplashKit.ColorPurple();

// Open a window
Window window = SplashKit.OpenWindow("Purple Example", 400, 400);

// Draw a filled circle on the screen using the color from above
SplashKit.ClearScreen();
SplashKit.FillCircle(myColor, 200, 200, 100);
SplashKit.RefreshScreen();

// Keep the window open for 5 seconds
SplashKit.Delay(5000);

// Close the window
SplashKit.CloseWindow(window);
